lib/absinthe/phoenix/channel.ex

```diff
@@ -56,6 +59,17 @@ defmodule Absinthe.Phoenix.Channel do
 
       {reply, socket} = run_doc(socket, query, config, opts)
 
+      ## Message ourselves to tell the server the client is ready to receive data now
+      ## Note, the below message will not be executed until this current message is finished
+      ## since they are part of the same GenServer.
+      handler =
+        socket.assigns
+        |> Map.get(:absinthe, %{})
+        |> Map.get(:opts, [])
+        |> Keyword.get(:context, %{handler: nil})
```

Comment:
why do we need `%{handler: nil}` here?

Reply:
Because you cannot do `Map.get(nil, :handler)` which will happen if the `context` is missing from `opts`

Reply:
This whole chain feels like it would be improved with use of Access since it is nil resilient:

```
socket.assigns[:absinthe][:opts][:context][:handler]
```